src/System.Net.Primitives/ref/System.Net.Primitives.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.10.0</AssemblyVersion>
```

Comment:
Shouldn't we bump this to 4.1? Or do we leave this as-is and only dump dir.props?  Since there are a bunch of APIs added since previous RTM, the contract should be at 4.1.

cc: @ericstj